CLAUDE.md

```diff
@@ -34,23 +34,35 @@ The repository uses `pyinvoke` for task automation with a modular task system:
 
 ## Common Development Commands
 
+### Thin environment
+
+```bash
```

Comment:
I usually run it directly in one of my tmux windows so it's already config.

Is this needed?